packages/vm/tests/tester/runners/GeneralStateTestsRunner.ts

```diff
@@ -116,6 +122,7 @@ async function runTestCase(options: any, testData: any, t: tape.Test) {
         await vm.runTx({ tx, block })
         execInfo = 'successful tx run'
       } catch (e: any) {
+        console.log(e)
```

Comment:
```suggestion
```

Can we remove this console log?

Reply:
We should ook in our eslint rules, you get a warning if it happens in /src but not if it is in /tests. (Which is annoying)

Reply:
I think Ryan turned it off for tests but that's an easy fix if so.

Reply:
> We should ook in our eslint rules, you get a warning if it happens in /src but not if it is in /tests. (Which is annoying)

Just to round this out.  The `no-console` lint rule is only off in the VM (and only in the test runner) since we have a lot of custom logging output there.  We could switch it all to use tape comments or something if it's important enough to turn the warning back on for this section of the code.

![image](https://user-images.githubusercontent.com/17355484/182203670-e47b1611-29f1-4d9c-bfd6-e9a9f98c65e8.png)